downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -61,6 +84,22 @@ public function main() {
 ```
 
 ### Improvements
+- Made the type node in module-level constant declarations optional. Both of the following cases are valid.
+    ```ballerina
+    const map<string> personA = {FirstName : "Emily", LastName : "Johnson"};
+    const personB = {FirstName : "David", LastName : "Thompson"};
```

Comment:
Use uppercase for constants. Field names should start with lowercase letters. Fix formatting.